src/Dungeon/Entrance.elm

```diff
@@ -55,6 +55,7 @@ toTile (A ( entranceType, pos )) =
     in
         Tile.toTile pos tileType
 
-equal: Entrance -> Entrance -> Bool
+
+equal : Entrance -> Entrance -> Bool
 equal (A e1) (A e2) =
-    snd e1 == snd e2
\ No newline at end of file
+    Tuple.second e1 == Tuple.second e2
```

Comment:
Should be e1Position, e2Position or when I turn this back into an alias, e1.position, e2.position